doc/cdi-populators.md

```diff
@@ -154,12 +154,12 @@ spec:
   dataSourceRef:
     name: ovirt-pop
     kind: OvirtVolumePopulator
-    apiGroup: forklift.konveyor.io
+    apiGroup: forklift.cdi.konveyor.io
```

Comment:
wouldn't `cdi.forklift.konveyor.io` be more appropriate?

Reply:
agree.

Reply:
done